client/components/search-result-list.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Stars from './stars';
+
+export default class SearchResultList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    let counter = 1;
```

Comment:
Instead of using a counter, you can use the `index` that `.map()` gives you.

```js
searchResults.map((result, index) => {
  const number = index + 1;
})
```